magento/devdocs#: Add info about setPaymentMethodAndPlaceOrder to GraphQL checkout tutorial (#4950)
magento/devdocs: guides/v2.3/graphql/tutorials/checkout/checkout-payment-method.md

}
```

There are two mutation queries in GraphQl which can be use to set the payment method for your order:

|Mutation|Description|
|--- |--- |
|`setPaymentMethodOnCart`|Sets the payment method for your order|

> |`setPaymentMethodAndPlaceOrder`|Sets the payment method and then immediately places your order. In this case ["Step 10. Place the order"]({{ page.baseurl }}/graphql/tutorials/checkout/checkout-place-order.md) can be skipped|

Reviewer: ```suggestion
|`setPaymentMethodAndPlaceOrder`|Sets the payment method and then immediately places your order. In this case ["Step 10. Place the order"]({{ page.baseurl }}/graphql/tutorials/checkout/checkout-place-order.html) can be skipped|
```

Author: Thank you, @keharper

Author: fixed by @keharper ;)

Reviewer: @atwixfirster I can't commit the change. I can only make a suggestion to fix it

Author: Not sure how but you fixed that bug. I've pulled the changes and detected what `md` extension has been replaced with `html`.

Please run tests now.

Thank you!

}
```

> ### Set payment method and place order {#setPaymentMethodAndPlaceOrder}

Reviewer 2: I've just realized that this mutation is not a part of 2.3.2 version. I would suggest splitting this PR into two separate PRs and making the PR with information about `setPaymentMethodAndPlaceOrder` mutation to the develop branch.

Author: I've created a separate PR https://github.com/magento/devdocs/pull/5000 to fix 
1. Remove `address_type` from `shipping_addresses` and `billing_addresses`.
2. Remove `label` from `selected_shipping_method`

Reviewer 2: Thank you. The only change left is to create a PR to the `develop` branch.

Reviewer: @rogyar I will reset the target branch for this PR into the `develop` branch immediately after the next time we sync `master` and `develop`.

> Use the `setPaymentMethodAndPlaceOrder` mutation to set the payment method for your order.

Reviewer 2: Please, add a clarification here 

```suggestion
Use the `setPaymentMethodAndPlaceOrder` mutation to set the payment method and place the order.
```

Author: good point @rogyar 

**fixed**

**Request**

```text
mutation {
  setPaymentMethodAndPlaceOrder(input: {
      cart_id: "{ CART_ID }"
      payment_method: {
          code: "checkmo"
      }
  }) {    
    order {
      order_id
    }
  }
}
```

**Response**

> If the operation is successful, the response contains the code of the selected payment method.

Author: order ID, @keharper

> If the operation is successful, the response contains the code of the selected payment method.

Author: **fixed**
